toolbox

```diff
@@ -1236,23 +1196,25 @@ list_images()
 )
 
 
-containers_get_details()
+get_containers()
```

Comment:
There's a reason why we first get the names, and then the rest of the details. :) Granted, it's a very obscure reason.

Since Podman's filters don't support using a logical `or`, we had to separately query the `com.redhat.component` and `com.github.debarshiray.toolbox` tagged images and merge the results into a single list.

*(I wonder if we can conjure up a logical `or` now that Podman filters support regular expressions.)*

Merging the lists using `uniq(1)` is tricky. We use attributes like `{{.Created}}` for each item, which can potentially change across two different Podman invocations if the item is new enough to be less than a minute old, and hence prevent `uniq(1)` from combining them into one entry.

That's why the names are first queried, merged, and then the attributes are read.

Now that we bumped our minimum Podman requirement to 1.4.0, it would've been nice if the regular expression support was also part of 1.4.0. Sadly, it isn't. It landed only in 1.5.0. Given that we still run into the occasional Podman regression (eg., https://github.com/containers/libpod/issues/3946) I am not comfortable hard bumping the baseline to 1.5.0. At least not yet. It would be nice to be able to keep using 1.4.0 on a best effort basis.

This might mean that we either figure out a way to detect the regular expression support at run-time and use it, or we fix this bug as part of our rewrite in Go because we'd just vendor in the necessary `libpod` version. I am leaning towards the latter option, myself.